src/App.js

```diff
@@ -0,0 +1,17 @@
+import './App.css';
+import { Routes, Route } from 'react-router-dom';
+import Categories from './components/Categories';
+import Home from './components/Home';
+
+function App() {
+  return (
+    <div className="App">
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/categories" element={<Categories />} />
+      </Routes>
+    </div>
+  );
+}
```

Comment:
- [ ] Good job so far 👏 I have seen that you created a NavBar component but you are not rendering it. I can say that It is important because It will allow someone to shift from home to the categories page very easily again because that's what NavBars are for 😃  Therefore, I would suggest that you import it here and render it right above the `<Routes>` so that everyone will shift easily while visiting the site 😉

In case you are wondering where you can put it 👇  But remember to import it please 😃 
```js
    <div className="App">
      <NavBar />
      <Routes>
        <Route path="/" element={<Home />} />
        <Route path="/categories" element={<Categories />} />
      </Routes>
    </div>
```